packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -584,6 +596,9 @@ export function attach(
       if (fiber != null) {
         fibersWithChangedErrorOrWarningCounts.add(fiber);
         updateMostRecentlyInspectedElementIfNecessary(id);
+        if (forceErrorForFiberIDs.has(id)) {
+          overrideError(id, false);
+        }
```

Comment:
This doesn't look like the right place to reset this map.

`clearErrorsAndWarnings` is about resetting console errors and warnings shown in the Components tree. It isn't related to the force-errors behavior.

I think these lines should be deleted.
```suggestion
```

I think you should add this cleanup to the `untrackFibers` method instead:
```js
if (forceErrorForFiberIDs.has(fiberID)) {
  forceErrorForFiberIDs.delete(fiberID);
  if (forceErrorForFiberIDs.size === 0) {
    setErrorHandler(shouldErrorFiberAlwaysNull);
  }
}
```